lib/index.d.ts

```diff
@@ -1243,6 +1243,8 @@ declare namespace sharp {
         sigma?: number | undefined;
     }
 
+    type ExtendMode = 'background' | 'copy' | 'repeat' | 'mirror';
```

Comment:
Sorry for the API naming confusion, which is my fault as I suggested it in the first place, but I think this type would be less confusing when named `ExtendWith`.

Reply:
No worries. It was clearly stated that it was just a proposal. Names of public API attributes are hard to change and now is the best time to think about it.